### PR TITLE
Add missing property "$distinct_id" for those converted from unidentified people

### PR DESCRIPTION
**Pull request by zihejia**

### CODE REVIEW

File: MixpanelDemo/MixpanelDemo.xcodeproj/xcshareddata/xcschemes/MixpanelDemo.xcscheme [hunk 1]
```diff
@@ -67,6 +67,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      enableThreadSanitizer = "YES"
```
Comment (jaredmixpanel) [inline MixpanelDemo/MixpanelDemo.xcodeproj/xcshareddata/xcschemes/MixpanelDemo.xcscheme:70]: intentional?

File: MixpanelDemo/MixpanelDemoTests/MixpanelBaseTests.swift [hunk 2]
```diff
@@ -22,12 +21,6 @@ class MixpanelBaseTests: XCTestCase, MixpanelDelegate {
         super.setUp()
         stubCalls()
         mixpanelWillFlush = false
-        
-        if let loginView = self.topViewController() as? LoginViewController {
```
Comment (jaredmixpanel) [inline MixpanelDemo/MixpanelDemoTests/MixpanelBaseTests.swift:26]: did you mean to get ride of this and make the storyboard changes as well?

**Review by jaredmixpanel — APPROVED** [review]
$distinct_id changes lgtm. just wondering about the unrelated changes.

File: MixpanelDemo/MixpanelDemoTests/MixpanelBaseTests.swift (hunk 2 above)
Comment (zihejia) [reply MixpanelDemo/MixpanelDemoTests/MixpanelBaseTests.swift:26]: yeah, for tests, we remove the need depending on the demo app

**Pull request merged**